ssg/build_remediations.py

```diff
@@ -259,6 +273,26 @@ def write_fixes(remediation_type, build_dir, output_path, fixes):
     tree.write(output_path)
 
 
+def write_fixes_to_dir(fixes, remediation_type, output_dir):
+    """
+    Writes fixes as files to output_dir, each fix as a separate file
+    """
+    try:
+        extension = REMEDIATION_TO_EXT_MAP[remediation_type]
+    except KeyError:
+        raise ValueError("Unknown remediation type %s." % remediation_type)
+
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
+    for fix_name in fixes:
```

Comment:
You can iterate over `.items()` and have `for fix_name, fix in fixes.items(): ...`, allowing you to avoid using `fix_contents` and `config`.